src/main/java/uk/gov/pay/webhooks/deliveryqueue/managed/SendAttempter.java

```diff
@@ -107,7 +108,6 @@ public void attemptSend(WebhookDeliveryQueueEntity queueItem) {
             );
             handleResponse(queueItem, DeliveryStatus.WILL_NOT_SEND, null, "Violates security rules", retryCount, start);
         } catch (Exception e) {
-            var responseTime = Duration.between(start, instantSource.instant());
```

Comment:
Ah, should this have been removed during a previous refactor at some point?

Reply:
Probably. I thought I might as well remove it now as the variable is not being used.